src/plugins/order_stash.coffee

```diff
@@ -176,31 +181,36 @@ class OrderStash
     font-style: italic;
     line-height: 13px;
     box-sizing: border-box;
+
+    transition: none;
+    text-decoration: none;
   }
 
-  #sa-order-stash-why:hover {
+  #sa-order-stash-plugin #sa-order-stash-why:hover {
     color: #666;
     border-bottom: 1px dashed #909090;
   }
 
-  #sa-order-stash-rationale {
+  #sa-order-stash-plugin #sa-order-stash-rationale {
     display: none;
     padding: 0 15px;
     text-align: left;
   }
 
-  #sa-order-stash-rationale p:first-child {
+  #sa-order-stash-plugin #sa-order-stash-rationale p:first-child {
     font-size: 14px;
   }
 
-  #sa-order-stash-rationale p {
+  #sa-order-stash-plugin #sa-order-stash-rationale p {
+    display: inline-block;
     margin-bottom: 23px;
     line-height: 17px;
     font-family: Verdana, Arial, sans-serif;
     font-size: 13px;
   }
 
-  #sa-order-stash-rationale p.sa-order-stash-privacy {
+  #sa-order-stash-plugin #sa-order-stash-rationale p.sa-order-stash-privacy {
+    font-style: italic;
```

Comment:
Same here it was already defined here: https://github.com/skroutz/analytics.js/pull/20/files#diff-7316f697d60038fadbc42b52a47928d0L104